logging/google/cloud/logging/handlers/handlers.py

```diff
@@ -15,27 +15,21 @@
 """Python :mod:`logging` handlers for Google Cloud Logging."""
 
 import logging
-
```

Comment:
This empty line stays (visual space indicating the import sections are different)